Assets/Scripts/Actor.cs

```diff
@@ -4,24 +4,23 @@
 
 public class Actor : MonoBehaviour
 {
+    public enum Team { Neutral, Blue, Red }; // Green, Purple, Orange
```

Comment:
You can think of Enums as a type, like a class. So you need to declare instances of them in your class to be able to reference them and change them.